### PR TITLE
GH-777: ui/ux updates to several pages

### PR DESCRIPTION
**Pull request by ksuhartono97**

### CODE REVIEW

**Review by bhryck — APPROVED** [review]

File: ui/src/components/about/StaffSection.js [hunk 1]
```diff
@@ -21,54 +21,61 @@ const StaffSection = (props) => {
...
+              <Box fontSize="md">
+                <Text color="black" paddingTop="1em">
                   {blurb.kim[0]}
                 </Text>
-                <Text color="white" fontSize="sm" paddingTop="1em">
+                <Text color="black" paddingTop="1em">
                   {blurb.kim[1]}
                 </Text>
-                <Text color="white" fontSize="sm" paddingTop="1em">
+                <Text color="black" paddingTop="1em">
```
Comment (tamarayustian) [inline ui/src/components/about/StaffSection.js:78]: i think u can make this array cleaner? something like this?
```
{blurb.kim.length && blurb.kim.map(((item) => {
return <Text color="black" paddingTop="1em"> {item} </Text>
})
```

File: ui/src/components/about/StrategySection.js [hunk 2]
```diff
@@ -2,157 +2,134 @@ import {
...
-        bg="#3182CE"
-        px={[0, 8]}
-        pt={[4, 6]}
-        pb={[4, 6]}
-        w="100%"
-        bgImage={`url('${process.env.PUBLIC_URL}/images/about-us/strategy.png')`}
-        bgPosition="center"
-        bgSize="cover"
-      >
+      <Box pt={[4, 6]} pb={[4, 6]} w="100%">
```
Comment (tamarayustian) [inline ui/src/components/about/StrategySection.js:15]: if both `pt` and `pb` is same then can just be `py`

File: ui/src/components/footer/FooterContainer.js [hunk 3]
```diff
@@ -1,98 +1,11 @@
...
-                Harvest Mission Community Church
-              </Text>
-              <Text fontWeight="bold" align="center">
-                (Hong Kong)
-              </Text>
-            </Box>
-            <ChurchAppDownloadButton />
-            <SoapAppDownloadButton />
-          </VStack>
+import FooterMobile from './FooterMobile';
```
Comment (tamarayustian) [inline ui/src/components/footer/FooterContainer.js:3]: wow i love this file

File: ui/src/components/footer/FooterDesktop.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+              <Link href="mailto:hongkong@hmcc.net" fontWeight="bold">
+                hongkong@hmcc.net
+              </Link>
+            </Box>
+          </VStack>
+          <VStack spacing="20">
+            <Button
+              variant="outline"
+              colorScheme="white"
+              as={Link}
```
Comment (tamarayustian) [inline ui/src/components/footer/FooterDesktop.js:52]: can this be new tab when going to hmcc.tv

File: ui/src/components/footer/FooterMobile.js [hunk 5]
```diff
@@ -0,0 +1,304 @@
...
+                            <Box as="span" flex="1" textAlign="left">
+                              Vision & Mission
+                            </Box>
+                          </AccordionButton>
+                        </h2>
+                      </AccordionItem>
+                      <AccordionItem border="none">
+                        <h2>
+                          <AccordionButton as={Link} href="/about-us#values">
+                            <Box as="span" flex="1" textAlign="left">
```
Comment (tamarayustian) [inline ui/src/components/footer/FooterMobile.js:118]: all these accordions is repetitive. can make it shorter and cleaner by mapping like an array
``` { array.map... <AccordionItem> ... </AccordionItem>)```

File: ui/src/components/footer/LinkGrid.js [hunk 6]
```diff
@@ -31,46 +27,46 @@ export function LinkGrid() {
...
-          <Link href="/about-us">
+          <Link href="/about-us#values">
             <Text>Our Values</Text>
           </Link>
-          <Link href="/about-us">
+          <Link href="/about-us#strategy">
             <Text>Our Strategy</Text>
           </Link>
-          <Link href="/about-us">
+          <Link href="/about-us#staff">
```
Comment (tamarayustian) [inline ui/src/components/footer/LinkGrid.js:62]: same as comment above

**Review by tamarayustian — CHANGES_REQUESTED** [review]

**Review by tamarayustian — APPROVED** [review]

**Pull request merged**